### ISSUE TITLE
Uncaught Error: More context objects were passed than there are dynamic segments for the route: contact

### ISSUE THREAD

**Issue by gosuto** [tonycoco/biznz#8]
Backend works great, having trouble with the frontend (Ember).

I've followed the tutorial exactly, index displays fine, but when clicking on "Contacts" I get:

"Uncaught Error: More context objects were passed than there are dynamic segments for the route: contact"

at ember.debug.js:44298

Also when I click on Home I get:

"Uncaught Error: Something you did caused a view to re-render after it rendered but before it was inserted into the DOM."

at ember.debug.js:39837

Any ideas?

**Comment by jcarlosteso** [comment]
Same here. Did you solve it?

**Comment by gosuto** [comment]
@jcarlosteso no, I bailed on the whole idea of Rails + Ember for now, decided our application wasn't front end heavy enough to go through the Ember learning curve (for now).
